pypechain/main.py

```diff
@@ -136,8 +140,9 @@ def parse_arguments(argv: Sequence[str] | None = None) -> Args:
     parser = argparse.ArgumentParser(description="Generates class files for a given abi.")
 
     parser.add_argument(
-        "abi_file_path",
-        help="Path to the abi JSON file or directory containing multiple JSON files.",
+        "abi_file_paths",
+        nargs="*",
```

Comment:
this is a required argument, right? so I think we would want `+` instead of `*` so it will throw an error if no arg is given.